ifrs17/Import/ImportScopeCalculation.ipynb

```diff
@@ -538,7 +538,10 @@
                 "\n{",
                 "\n    static ApplicabilityBuilder ScopeApplicabilityBuilder(ApplicabilityBuilder builder) =>",
                 "\n        builder.ForScope<NominalCashflow>(s => s.WithApplicability<CreditDefaultRiskNominalCashflow>(x => x.Identity.AmountType == AmountTypes.CDR && x.Identity.Id.AocType == AocTypes.CF)",
-                "\n                                                .WithApplicability<AllClaimsCashflow>(x => x.Identity.AmountType == AmountTypes.CDR));",
+                "\n                                                .WithApplicability<AllClaimsCashflow>(x => x.Identity.AmountType == AmountTypes.CDR)",
+                "\n                                                .WithApplicability<ProjectionBopNominalCashflow>(x => x.GetStorage().GetShift(x.Identity.Id.ProjectionPeriod) >= MonthInAYear",
```

Comment:
I just realize that this implementation would not compute the correct thing for AmountType.CDR : previous applicability. 
What I think would work is to move the applicability to the ReferenceAocStep which is used to modify the identity in this scope.

Reply:
@dcolleoni Do you mean computing PresentValue.Value with the ReferenceAocStep scope directly?

Reply:
Done. Introduced changes inside the ReferenceAocStep scope.